admin/spec/back-end/controllers/accessibility-statement.spec.js

```diff
@@ -5,7 +5,7 @@
 const httpMocks = require('node-mocks-http')
 
 describe('accessibility-statement page controller', () => {
-  it('should render the initial accessibility-statement page', async (done) => {
```

Comment:
the rest of the PR is now just removing the done param and calls which were necessary to support async in jasmine.  there is a lot of it.